src/renderers/dom/shared/devtools/ReactDOMUnknownPropertyDevtool.js

```diff
@@ -66,36 +66,56 @@ if (__DEV__) {
       null
     );
 
-    warning(
-      registrationName == null,
-      'Unknown event handler property %s. Did you mean `%s`? %s',
-      name,
-      registrationName,
-      formatSource(source)
-    );
-  };
-
-  var formatSource = function(source) {
-    return source ? `(${source.fileName.replace(/^.*[\\\/]/, '')}:${source.lineNumber})` : '';
+    if (standardName != null) {
+      warning(
+        standardName == null,
+        'Unknown DOM property %s. Did you mean %s? %s',
```

Comment:
nit: don't need a space before the addendum – can you remove it (from all three)?